src/auro-flightline.js

```diff
@@ -49,17 +59,18 @@ class AuroFlightline extends LitElement {
   render() {
     const ONE = 1,
      classes = {
-      'nonstop': !this.children.length,
+      'nonstop': !this.children.length && !this.canceled,
       'single': this.children.length === ONE,
       'multiple': this.children.length > ONE,
+      'canceled': this.canceled,
       'slot-container': true,
     };
 
     return html`
       <div class="${classMap(classes)}">
-        <slot></slot>
-        ${this.children.length > ONE ? html`
-          <auro-flight-segment iata="${this.children.length} stop${this.children.length > ONE ? 's' : ''}"></auro-flight-segment>
+        ${!this.canceled ? html` <slot></slot>`: html`<span></span>`}
```

Comment:
```suggestion
        ${!this.canceled ? html` <slot></slot>`: html``}
```

I am not seeing any need for these spans? If that is true, let's delete these and then this PR is perfect!

Reply:
Waiting on this last update and this can be merged.